src/GithubAPI.js

```diff
@@ -36,6 +36,6 @@ export default class GithubAPI {
       user  : `/users/${key}`
     }[type];
     const url = "https://api.github.com" + path;
-    return execSync("curl -H 'Authorization: token " + process.env.GITHUB_AUTH + "' --silent " + url)
+    return execSync("curl -H 'Authorization: token " + process.env.GITHUB_AUTH + "' --silent --globoff " + url)
```

Comment:
This is to allow fetching url that may contain `[ ]` (e.g. `greenkeeper[bot]`)

http://stackoverflow.com/questions/8333920/passing-a-url-with-brackets-to-curl